docs/examples/plot_diffuse_aoi_correction.py

```diff
@@ -0,0 +1,118 @@
+"""
+Diffuse IAM Calculation
+=======================
+
+Integrating an IAM model across angles to determine the overall reflection
+loss for diffuse irradiance.
+"""
+
+# %%
+# The fraction of light reflected from the front of a module depends on the
+# angle of incidence (AOI) of the light compared to the panel surface.  The
+# greater the AOI, the larger the reflected fraction is.  The fraction of
+# transmitted light to incident light is called the incident angle modifier
+# (IAM).  Several models exist to calculate the IAM for a given incidence
```

Comment:
>The fraction of transmitted light to incident light is called the incident angle modifier (IAM).

Is this what Marion says? Because strictly speaking, I don't think this is true. At normal incidence, there might be 4% internal reflection, meaning transmitted light is only 96%. You can derive this directly from [Fresnel's equation at normal incidence](https://en.wikipedia.org/wiki/Fresnel_equations#Normal_incidence):

```
R = ((n1 - n2)/(n1 + n2))**2
```

So if you look at the [Sandia Array Performance model](https://prod-ng.sandia.gov/techlib-noauth/access-control.cgi/2004/043535.pdf) the IAM or f2 is defined as the following ratio:

```
f2_aoi = Isc(aoi=aoi) / Isc(aoi=0)
```

Reply:
You're right, sloppy and incorrect wording on my part.  Marion's definition matches yours:

> The AOI correction factor is the in-plane irradiance reaching the PV cells divided by the in-plane irradiance reaching the PV cells when the AOI is normal to the PV module surface.

I'll update with your suggestion, although I'll change `fraction` to `ratio`.  Not sure why I used `fraction` originally, it reads a bit awkwardly.